auth/auth/templates/users.html

```diff
@@ -1,53 +1,76 @@
-    <button>
-      Create
-    </button>
-  </form>
+<div class='w-full md:w-2/5 flex flex-col md:flex-row justify-between'>
+  <div>
+    <h1 class='text-2xl font-light mb-4'>Create User</h1>
+    <form action="{{ base_path }}/users" method="POST">
+      <div class='flex-col space-y-2'>
+        <div class='flex justify-between space-x-4'>
+          <span>Username</span>
+          <input class='border rounded-sm p-1 text-sm' required name="username" />
+        </div>
+        <div class='flex justify-between space-x-4'>
+          <span>Login ID</span>
+          <input class='border rounded-sm p-1 text-sm' name="login_id" />
+        </div>
+        <div><input type="checkbox" name="is_developer" value="1" /> Developer</div>
+        <div><input type="checkbox" name="is_service_account" value="1" /> Service Account</div>
+        <input type="hidden" name="_csrf" value="{{ csrf_token }}" />
+        <div class='flex justify-end'>
+          {{ submit_button('Create') }}
+        </div>
+      </div>
+    </form>
+  </div>
+  <div>
+    <h1 class='text-2xl font-light mb-4'>Delete User</h1>
+    <form action="{{ base_path }}/users/delete" method="POST">
+      <div class='flex-col space-y-2'>
+        <div class='flex justify-between space-x-4'>
+          <span>User ID</span>
+          <input class='border rounded-sm p-1 text-sm' required name="id" />
+        </div>
+        <div class='flex justify-between space-x-4'>
+          <span>Username</span>
+          <input class='border rounded-sm p-1 text-sm' required name="username" />
+        </div>
+        <input type="hidden" name="_csrf" value="{{ csrf_token }}" />
+        <div class='flex justify-end'>
+          {{ danger_button('Delete') }}
+        </div>
+      </div>
+    </form>
+  </div>
+</div>
 
-  <h1>Delete User</h1>
-  <form action="{{ base_path }}/users/delete" method="POST">
-    <div>User ID: <input name="id" /></div>
-    <div>Username: <input name="username" /></div>
-    <input type="hidden" name="_csrf" value="{{ csrf_token }}"/>
-    <button  class="dangerous">
-      Delete
-    </button>
-  </form>
-
-  <h1>Users</h1>
-  <table class="data-table">
-    <thead>
-      <tr>
-        <th>ID</th>
-        <th>Username</th>
-        <th>Login ID</th>
-        <th>Hail Identity</th>
-        <th>State</th>
-        <th>Developer</th>
-        <th>Robot</th>
-      </tr>
-    </thead>
-    <tbody>
-      {% for user in users %}
-      <tr>
-        <td class="numeric-cell">{{ user['id'] }}</td>
-        <td>{{ user['username'] }}</td>
-        <td>{{ user['login_id'] }}</td>
-        <td>{{ user['hail_identity'] }}</td>
-        <td>{{ user['state'] }}</td>
-        <td class="numeric-cell">{{ user['is_developer'] }}</td>
-        <td class="numeric-cell">{{ user['is_service_account'] }}</td>
-      </tr>
-      {% endfor %}
-    </tbody>
-  </table>
+<br />
+<h1 class='text-2xl font-light mb-4'>Users</h1>
+<table class="table-auto">
```

Comment:
i'd like this table to match the one on the batches page if possible, spanning the full page, with the white background and lines between rows, and with the headers aligned the same way as the cell text (all left-aligned except center for both the header and text in the first row)

Reply:
Done! I centered the check marks and Xs but let me know if you'd rather them be left aligned.